packages/forms-web-app/src/controllers/examination/file-upload/fileManagement.js

```diff
@@ -22,8 +22,16 @@ const deleteFileOnDisk = async (file) => {
 	await fs.unlinkSync(fileToDeletePath);
 };
 
+const iterateDeleteFileOnDisk = async (files) => {
+	for (const filesKey of files) {
+		const fileToDeletePath = fileUploadPath(filesKey.uniqueFileName);
+		await fs.unlinkSync(fileToDeletePath);
```

Comment:
No real need to have this as sync other than to allow us to throw an error if the delete fails.